tests/integration/features/istio-jwt/istio_jwt_scopes.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Exposing two endpoints with Istio JWT authorization strategy with scopes
+
+  Scenario: IstioJWTScopes: Calling an endpoint secured with JWT with a valid token
```

Comment:
Is it possible to have the scenario description more specific? Something like "Calling endpoints secured with JWT and enforcing JWT scope claims"?